tests/finland/test_ekirjasto.py

```diff
@@ -311,19 +311,12 @@ def test_secrets(
     ):
         provider = create_provider()
 
-        # Secrets are not set, so this will fail.
```

Comment:
The case of secrets not being set are also covered by the test cases following this one, so removed from here to fix mypy errors.